### ISSUE TITLE
Align JSKOS ConceptScheme properties with MOD vocabulary

### ISSUE THREAD

**Issue by nichtich** [gbv/jskos#104]
The [MOD vocabulary](https://www.isibang.ac.in/ns/mod/) collects classes and properties for description of ontologies. An ontology (or any other controlled vocabulary) in JSKOS is an instance of [JSKOS ConceptScheme](https://gbv.github.io/jskos/jskos.html#concept-schemes). BARTOC collects metdata about these vocabularies and exposes the metadata in JSKOS format, with some additional properties (which might become part of JSKOS specification): `EXAMPLES`, `ACCESS`, `ADDRESS`, `CONTACT`, `FORMAT`. We also don't fully use [JSKOS Distribution](https://gbv.github.io/jskos/jskos.html#distributions) yet.

Anyway, I comparison of MOD and JSKOS/BARTOC could improve the data model and result in an another mapping from JSKOS to RDF (see current [JSON-LD context](https://gbv.github.io/jskos/jskos.html#json-ld-context)).

**Comment by biswanathdutta** [comment]
@nichtich yes, we can do a mapping between JSKOS/BARTOC and MOD. As suggested above, some of the properties from MOD proposition may also be adopted to the BARTOC catalogue. Let us discuss this.

**Comment by nichtich** [comment]
Looks like MOD is used in AgroPortal, so alignment is needed in NFDI to integrate data from BARTOC and from AgroPortal. Current repository is https://github.com/FAIR-IMPACT/MOD, currently being worked on.
